integration-tests/jest/1-run-workflow-on-snapshot.integration-test.js

```diff
@@ -2,4 +2,4 @@ const { registerTest } = require('./jest-utils')
 const { testRunWorkflowOnSnapshot } = require('../tests/run-workflow-on-snapshot')
 
 
-registerTest(testRunWorkflowOnSnapshot)
+registerTest(testRunWorkflowOnSnapshot, 1)
```

Comment:
I haven't looked at this enough to know why it fails, but I've always seen the screenshot be on the workflow configuration screen. Everything there should be in our control, so I feel like we really should be able to fix that. (It might occasionally fail because the workflow took too long, same as `run-workflow`).